i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx

```diff
@@ -22,6 +22,32 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
 
 ## Centreon Web
 
+### 23.10.6
+
+Release date: `January 12, 2024`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [API] Added dashboard shares to dashboards list.
+
+</details>
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Authentication] Fixed hardcoded email value that block SSO authentication with OpenId Connect when parameter is not email.
+- [Configuration] Fixed error message displayed in configuration generation after upgrading to 23.04 version.
+- [Event Logs] Fixed period in CSV export.
+- [Install] Fixed installation when database user already exists.
+- [Installation] Fixed issue adding twice poller ID in Centreon gorgone configuration.
+- [My Account] Fixed account properties edition for local user only.
```

Comment:
```suggestion
- [My Account] Made the name, login and password fields read-only for non-local users so as to prevent issues where users who had edited these properties were locked out of the platform.
```